redact-wasm/package.json

```
@@ -0,0 +1,69 @@
+{
+  "name": "@arcjet/redact-wasm",
+  "version": "1.0.0-alpha.21",
```

```suggestion
  "version": "1.0.0-alpha.22",
```
We're on a new version now